src/components/autoFocuser.js

```diff
@@ -0,0 +1,67 @@
+define(["focusManager"], function (focusManager) {
+    "use strict";
+
+    /**
+     * Previously selected element.
+     */
+    var activeElement;
+
+    /**
+     * Start AutoFocuser
+     */
+    function enable() {
+
+        window.addEventListener("focusin", function(e) {
+            activeElement = e.target;
+        });
+
+        console.log("AutoFocuser enabled");
+    }
+
+    /**
+     * Set focus on a suitable element, taking into account the previously selected.
+     */
+    function autoFocus(container) {
+
+        container = container || document.body;
+
+        var candidates = [];
+
+        if (activeElement) {
+            // These elements are recreated
+            if (activeElement.classList.contains("btnPreviousPage")) {
+                candidates.push(container.querySelector(".btnPreviousPage"));
+                candidates.push(container.querySelector(".btnNextPage"));
+            } else if (activeElement.classList.contains("btnNextPage")) {
+                candidates.push(container.querySelector(".btnNextPage"));
+                candidates.push(container.querySelector(".btnPreviousPage"));
+            } else if (activeElement.classList.contains("btnSelectView")) {
+                candidates.push(container.querySelector(".btnSelectView"));
+            }
+
+            candidates.push(activeElement);
+        }
+
+        candidates = candidates.concat(Array.from(container.querySelectorAll(".btnResume")));
+        candidates = candidates.concat(Array.from(container.querySelectorAll(".btnPlay")));
```

Comment:
Can we avoid the array creation here somehow?

Reply:
Do you mean `Array.from`?
Or `candidates` itself?

Reply:
Array.from

Reply:
Currently, there is one `btnResume` on `itemdetails.html`, so I can change it to
```
candidates.push(container.querySelector(".btnResume"));
```
But there is two `btnPlay`. One of them (the first one) is hidden in TV layout in `itemBackdrop`.

Are you worried about performance?